modules/config/escalations.tf

```diff
@@ -15,22 +15,18 @@ resource "opsgenie_escalation" "this" {
       notify_type = try(rules.value.notify_type, "default")
       delay       = try(rules.value.delay, 0)
 
-      dynamic "recipient" {
-        for_each = try(rules.value.recipients, [])
-
-        content {
-          type = recipient.value.type
-
-          id = lookup(recipient.value, "id", null) != null ? recipient.value.id : (
-            recipient.value.type == "team" ? opsgenie_team.this[recipient.value.team_name].id : (
-              recipient.value.type == "user" ? try(opsgenie_user.this[recipient.value.user_name].id, data.opsgenie_user.this[recipient.value.user_name].id) : (
-                recipient.value.type == "schedule" ? opsgenie_schedule.this[recipient.value.schedule_name].id : (
-                  null
-                )
+      recipient {
```

Comment:
Line 11 from `for_each = try([each.value.rules], [])` to `for_each = try(each.value.rules, [])`